src/styles/ManageOwners.css

```diff
@@ -165,6 +165,10 @@
 
       &.add {
         height: 42px;
+        background: transparent;
+        color: #454196;
```

Comment:
Maybe we can define `colors.css` that defines all colors used in the app and import from there into the other CSS files. I mean centralize all colors so is easier to change colors in the future, what do you think?

Reply:
I agree, but that task would take more time an effort, it should have its own PR